lib/data/service/lesson_service.dart

```diff
@@ -26,4 +27,32 @@ class LessonServices {
 
     return bookList.map((book) => BookModel.fromJson(book as Map<String, dynamic>)).toList();
   }
+
+  Future<List<ChapterModel>> fetchChapters(int bookId) async {
+    final HttpLink httpLink = HttpLink('https://api-preprod.lelivrescolaire.fr/graph');
+
+    final GraphQLClient client = GraphQLClient(
```

Comment:
Since you're using a cache, you probably don't want to create a new instance of the client for each request.